src/app/families/FamilyDeliveries.ts

```diff
@@ -83,6 +83,9 @@ export class FamilyDeliveries extends IdEntity {
         allowApiUpdate: Roles.admin
     });
     quantity = new QuantityColumn(this.context, { caption: getLang(this.context).quantity, allowApiUpdate: Roles.admin, dataControlSettings: () => ({ width: '100', inputType: 'number' }) });
+    isLargeQuantity() {
```

Comment:
should restrict this to you.
The large quantity doesn't always make sense (I have a team that uses the quantity for the cash amount they should charge the recipient)

Reply:
I will restrict it to MLT. The version I based on did not have the new isSystemForMLT() - I will update it now